relayer/pkg/chainlink/ocr2/transmissions_cache.go

```diff
@@ -27,11 +26,11 @@ type transmissionsCache struct {
 	stop, done chan struct{}
 
 	reader *contractReader
-	cfg    starknet.Config
+	cfg    config.Config
```

Comment:
Nesting many packages we get types that read like `config.Config` which is not ideal. We should iterate on how to structure this better.